build.sbt

```diff
@@ -147,6 +147,12 @@ lazy val casper = (project in file("casper"))
   .settings(inConfig(SlowcookerTest)(org.scalafmt.sbt.ScalafmtPlugin.scalafmtConfigSettings))
   .settings(
     name := "casper",
+    scalacOptions ++= Seq(
```

Comment:
This `Seq` should be extracted into a constant, as it's used in multiple sub-modules

Reply:
👍